packages/server/core/utils/apiParams.ts

```diff
@@ -0,0 +1,5 @@
+export const apiDefaultHeaders = () => ({
+  headers: {
+    'content-type': 'application/json',
+  },
+})
```

Comment:
Почему бы не сделать просто объект?
```suggestion
export const apiDefaultHeaders = {
  headers: {
    'content-type': 'application/json',
  },
}
```

Reply:
исправлено